diagnostic.md

```diff
@@ -49,7 +53,9 @@ will take the base color and lighten it by the percentage indicated by mixing it
 with white. Write the code to tint `$cornflower` by 20%.
 
 ```scss
-// your response here
+@function tint($cornflower, 20%) {
+  @return mix($white, $cornflower, 20%);
```

Comment:
Correct. That is the function itself. The way you call it is

`tint($cornflower, 20);